libraries/core_libs/consensus/include/pbft/step/step.hpp

```diff
@@ -95,11 +30,18 @@ class Step {
    */
   size_t placeVote_(blk_hash_t const& blockhash, PbftVoteTypes vote_type, uint64_t round);
 
+  /**
+   * @brief Terminate the soft voting value of the PBFT block hash
+   * @return true if terminate the vote value successfully
+   */
+  bool giveUpSoftVotedBlock_();
```

Comment:
as this is major refactor, lets not use _ in funciton names anymore just because it was there before